src/app/mini-games/pages/mini-games-page/mini-games-page.component.html

```diff
@@ -0,0 +1,60 @@
+<div class="flex-row">
```

Comment:
@suggestion
You could create array with games and iterate through it using ```*ngFor```

Reply:
done